src/api/middleware/files.js

```diff
@@ -0,0 +1,110 @@
+const HTTPStatus = require("http-status-codes");
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const { MulterError } = require("multer");
+const multerConfig = require("../../config/multer");
+const { ErrorTypes } = require("./errorHandler");
+const cloudinary = require("cloudinary").v2;
+const env = require("../../config/env");
```

Comment:
```suggestion
const config = require("../../config/env");
```
To follow the standard in other files